luaunit.lua

```diff
@@ -175,6 +175,22 @@ local function sortedPairs(tbl)
 end
 M.private.sortedPairs = sortedPairs
 
+math.randomseed(os.clock()*100000000000)
+
+local function randomizeTable( t )
+    -- Shuffle the table t
+    local rand = math.random 
+    assert( t, "shuffleTable() expected a table, got nil" )
+    local iterations = #t
+    local j
+    
+    for i = iterations, 2, -1 do
+        j = rand(i)
+        t[i], t[j] = t[j], t[i]
+    end
```

Comment:
We could cut down on `local`s here. I don't think that "caching" `math.random` makes much of a difference, and `iterations` is only used to initialize the loop variable, so I would do away with the `local rand` and suggest

```lua
    for i = #t, 2, -1 do
        local j = math.random(i)
        t[i], t[j] = t[j], t[i]
    end
```